.gitignore

```diff
@@ -1,4 +1,5 @@
 vendor/
 dist/
 config.json
-coverage.txt
\ No newline at end of file
+coverage.txt
+/matterpoll-emoji
```

Comment:
I added this because if you run `go build .` you get a binary named `matterpoll-emoji` which shouldn't be checked into the repository since it's a build artifact and is platform/architecture specific. I don't install go on production servers so I have to build a binary which is probably going to be more common than using Go to run main.go so I thought it was a logical change for most people.